### PR TITLE
 last active tab drag shift

### PR DESCRIPTION
**Pull request by prasook-jain**
- fix: last active tab drag shift
- lockOffset of tab drag element
- version update to 0.5.19

### CODE REVIEW

**Comment by prasook-jain** [comment]
test this please

File: src/components/Tabs/types.ts [hunk 1]
```diff
@@ -31,6 +33,11 @@ export interface TabsProps extends BaseComponentProps {
   addButtonElement?: React.ReactElement
   /** Custom Color (Valid color or theme color provided by knit-ui)*/
   customColor?: string
+  /**
+   * As lockToContainerEdges is set `true`, current lockOffset is 0 for react-sortable-hoc, here use can pass
+   * a props & define how much flexible you want your lock offset.
+   */
```
Comment (anshumanv) [inline src/components/Tabs/types.ts]:
```suggestion
  /**
   * As lockToContainerEdges is set `true`, current lockOffset is 0 for react-sortable-hoc, here user can pass
   * a prop & define how much flexibility user wants for lock offset.
   */
```

Comment (anshumanv) [reply src/components/Tabs/types.ts]: Isn't the default value 50%?

Comment (prasook-jain) [reply src/components/Tabs/types.ts]: yes my bad, but 50% is not working out for us. so set it to 100, which is 100px.

Comment (prasook-jain) [reply src/components/Tabs/types.ts]: making changes in comment

**Pull request merged**